src/plots/cartesian/layout_attributes.js

```diff
@@ -310,6 +310,14 @@ module.exports = {
             'If *B*, 1B.'
         ].join(' ')
     },
+    separatethousands: {
+        valType: 'boolean',
+        dflt: false,
+        role: 'style',
+        description: [
+            'even 4-digit integers are separated if true'
```

Comment:
Can make this description a little bit more like the other descriptions found in this file? For instance,

- Make sure the description starts with a capital letter
- We prefer starting with `If <something>, ...` rather than ending with  `... , if <something>`

These descriptions make up our [schema](https://plot.ly/javascript/reference/) pages, so we need to be as clear as possible.